modules/jt-comparison-of-jenkins-openshift-pipelines-execution-models.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assembly:
+//
+// jenkins/migrating-from-jenkins-to-openshift-pipelines.adoc
+
+[id="jt-comparison-of-jenkins-openshift-pipelines-execution-models_{context}"]
+= Comparison of Jenkins and OpenShift Pipelines execution models
+
+Jenkins and OpenShift Pipelines offer similar functions but are different in architecture and execution. This section outlines a brief comparison of the two execution models.
+
+.Comparison of execution models in Jenkins and OpenShift Pipelines
+[cols="1,1",options="header"]
+|===
+|Jenkins|OpenShift Pipelines
+|Jenkins has a control node. Jenkins executes pipelines and steps centrally, or orchestrates jobs running in other nodes.|OpenShift Pipelines is serverless and distributed, and there is no central dependency for execution.
+|The containers are launched by the control node through the pipeline.|OpenShift Pipelines adopts a 'container-first' approach, where every step is executed as a container running in a pod (equivalent to nodes in Jenkins).
+|Extensibility is achieved using plugins.|Extensibility is achieved using tasks in Tekton Hub, or by creating custom tasks and scripts.
```

Comment:
```suggestion
|Extensibility is achieved using plugins.|Extensibility is achieved by using tasks in Tekton Hub or by creating custom tasks and scripts.
```
- Should `Tekton`  here be changed to `OpenShift Pipelines`?

Reply:
Thanks. Updated. In this case, Tekton Hub is a site name that does not need to be changed. There is no equivalent hub for OpenShift Pipelines.